src/zoid/buttons/component.jsx

```diff
@@ -63,6 +69,29 @@ export const getButtonsComponent : () => ButtonsComponent = memoize(() => {
                     } }
                 />
             ).render(dom({ doc }));
+
+            // CPL stands for Consumer Perceived Latency
+            logLatencyInstrumentationPhase({ buttonSessionID, phase: 'buttons-first-render-end' });
```

Comment:
Ideally zoid would provide an event for knowing when prerender is complete. I added a feature request to zoid for adding this: https://github.com/krakenjs/zoid/issues/402